src/libs/actions/Session.js

```diff
@@ -433,5 +524,11 @@ export {
     clearSignInData,
     cleanupSession,
     clearAccountMessages,
+    setSessionLoadingAndError,
+    updateSessionAuthTokens,
     validateEmail,
+    authenticatePusher,
+    throttledReauthenticatePusher,
```

Comment:
I don't think we need to export a method called `throttledReauthenticatePusher`, that can remain an implementation detail specific to this file imo:

```suggestion
    reauthenticatePusher: throttledReauthenticatePusher,
```

Reply:
Ah yeah I think I have a good way to solve this. The suggested change isn't valid export syntax - but I can just move the body of `reauthenticatePusher` into `_.throttle()`.